sources/Makefile

```diff
@@ -222,3 +222,18 @@ connect-qa-db:
 	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5656."
 	@oc project 0c27fb-test
 	@oc port-forward services/patroni-master 5656:5432 
+
+backend-unit-tests:
+	@cd ./packages/backend && npm i && npm run test:cov
+
+e2e-tests-api:
+	@echo "+\n++ Make: Starting containers for API E2E tests...\n+"
+	@docker-compose -f docker-compose.yml run --name api-e2e-tests --rm api npm run test:e2e:api
```

Comment:
Great way to run tests with --rm ❤️

In future should we change local-api-test to follow the same way?

Reply:
I will remove `local-api-test`.
We can use the below ones from now on:
- `make backend-unit-tests`
- `make e2e-tests-api`
- `make e2e-tests-workers`
- `make e2e-tests-queue-consumers`